src/NCronJob/Observer/ExecutionProgress.cs

```diff
@@ -0,0 +1,40 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace NCronJob.Experimental;
```

Comment:
I would also remove the namespace here. Not really discoverable and the attribute is enough (IMHO)

Reply:
Fixed